tools/ci_build/github/windows/run_OpenCppCoverage.ps1

```diff
@@ -72,7 +72,9 @@ RunTest $mlas_test @() ("binary:" + (Join-Path $buildDir "onnxruntime_mlas_test.
 
 # Lotus unit tests
 # need to copy the tvm.dll, since it is not in the buildDir path
-Copy-Item -Path $BuildRoot\Debug\external\tvm\Debug\tvm.dll -Destination $buildDir
```

Comment:
this is only copied if we have --use_tvm build option. Will command fail if this option is not present? E.g. in NuGet packaging pipeline, this option is not present.

Reply:
The copy is made conditional now, so it won't fail if --use_tvm is not used.